source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -1,3 +1,131 @@
+.. _error_handling:
+
 ==============
 Error handling
 ==============
+
+oneDAL error handling relies on the  mechanism of C++ exceptions. If an error
+occurs, it shall be propagated at the point of a function call where it can be
+caught using standard C++ error handling mechanism.
+
+Exception classification
+========================
+
+Exception classification in onDAL is aligned with C++ STL classification. oneDAL
+shall introduce abstract classes defining the base for for hierarchy of
+exception classes. Сoncrete exception classes are derived from respective STL
+exception classes. oneDAL library shall throw exceptions represented with
+concrete classes.
```

Comment:
specific

Reply:
non-abstact? concrete seems to be the term from OOP, are there others? I don't think "specific" works here